Wellthier/View Controllers/WorkoutFeedViewController.m

```diff
@@ -160,11 +142,16 @@ - (NSInteger)tableView:(UITableView *)tableView numberOfRowsInSection:(NSInteger
 }
 
 - (UITableViewCell *)tableView:(UITableView *)tableView cellForRowAtIndexPath:(NSIndexPath *)indexPath {
+    UITapGestureRecognizer *tapGesture = [[UITapGestureRecognizer alloc]
+      initWithTarget:self action:@selector(handleTap:)];
+    tapGesture.delegate = self;
     PostCell *cell = [self.tableView dequeueReusableCellWithIdentifier:@"PostCell"];
     Post *post = self.arrayOfPosts[indexPath.row];
     cell.detailPost = post;
     [cell setPostDetails:cell.detailPost];
     cell.toProfileHiddenButton.tag = indexPath.row;
+    [cell.postImageView addGestureRecognizer:tapGesture];
```

Comment:
This can add multiple gesture recognizer to same cell postImage view. I would recommend to add this gestureReconizer in PostCell class.

You can then add some short of protocol/delegate to send communication back from PostCell class to your viewController something like

- (void)didTapImageView:(NSInteger)imageViewTag; 

Then you can continue finding your tableviewcell/image view in this method.